lib/stencil-push.utils.js

```diff
@@ -152,6 +151,14 @@ utils.promptUserToDeleteThemesIfNecessary = (options, callback) => {
         return async.nextTick(callback.bind(null, null, options));
     }
 
+    if (options.deleteOldest) {
+        const oldestTheme =  options.themes
+        .filter(theme => theme.is_private)
+        .map(theme => ({name:theme.name, uuid: theme.uuid, updated_at : theme.updated_at = new Date(theme.updated_at).valueOf()}))
```

Comment:
Is the `name` here just for inspection convenience? It looks like you're only using the `uuid` and the `updated_at`.

Also, is the `theme.updated_at = ` part necessary?

Reply:
Yeah the name can likely be removed, I used it to check against what is in the control panel to make sure it is getting the oldest theme.

Will also remove the `theme.updated_at = ` assignment, not sure why I wrote that.